app/client/src/components/pages/LocationMap/MapFunctions.js

```diff
@@ -824,3 +824,26 @@ export function gradientIcon({ id, stops }) {
     </table>
   );
 }
+
+// Gets the highlight symbol styles based on the provided geometry.
+export function getHighlightSymbol(geometry, highlightOptions) {
+  let symbol = { color: highlightOptions.color };
+  if (geometry.type === 'polyline') {
+    symbol['type'] = 'simple-line';
+    symbol['width'] = 5;
+  } else if (geometry.type === 'polygon') {
+    symbol['type'] = 'simple-fill';
+    symbol['outline'] = {
+      color: highlightOptions.color,
+      width: 2,
+    };
+  } else if (geometry.type === 'point' || geometry.type === 'multipoint') {
+    symbol['type'] = 'simple-marker';
+    symbol['outline'] = {
+      color: highlightOptions.color,
+      width: 2,
+    };
+  }
```

Comment:
I'd suggest passing in the "color" property of the "highlightOptions" object directly, as that's all you're using in this function (see corresponding change in `hooks.js` file as well, where this function is used).

```suggestion
export function getHighlightSymbol(geometry, color) {
  let symbol: Object = { color };
  if (geometry.type === 'polyline') {
    symbol.type = 'simple-line';
    symbol.width = 5;
  } else if (geometry.type === 'polygon') {
    symbol.type = 'simple-fill';
    symbol.outline = { color, width: 2 };
  } else if (geometry.type === 'point' || geometry.type === 'multipoint') {
    symbol.type = 'simple-marker';
    symbol.outline = { color, width: 2 };
  }
```

(NOTE: generic "Object" type annotation on the "symbol" variable only to suppress Flow errors)

Reply:
Sounds good. I went ahead and made this change.